katalog/grafana/README.md

```diff
@@ -3,63 +3,65 @@
 Grafana is an open source data visualization and graph composer platform for
 numeric time-series data with Prometheus integration.
 
-
 ## Image repository and tag
 
-* Grafana image: `grafana/grafana:6.6.2`
-* Grafana repository: https://github.com/grafana/grafana
-* Grafana documentation: https://docs.grafana.org
-
+-   Grafana image: `grafana/grafana:6.6.2`
```

Comment:
Why the `* ` has been disappeared?

Reply:
because is a duplication of the `-` list in markdown.